cybox/common/vocabs.py

```diff
@@ -6,13 +6,58 @@
 from cybox.common import PatternFieldGroup
 from cybox.utils import normalize_to_xml, denormalize_from_xml
 
-INVALID_XSI_TYPE = "bad_xsi_type"
+
+class VocabField(cybox.TypedField):
```

Comment:
As for the implementations, I just followed what I did for python-stix. If we change it here then we should probably change it for python-stix, and I don't know if that will just make the imports messy when trying to support backwards compatibility. Personally, i'd rather keep them together than break them apart as the imports seem a bit more intuitive from an end-user perspective.